builder.md

```diff
@@ -1,42 +1,47 @@
 
-A base16 builder is a tool that builds application specific themeing configurations. It does this by using base16 scheme files (containing a collection of colors) and base16 template files (instructions concerning how to build the application specific files).
+A theme builder is a tool that builds application specific themeing configurations. It does this by using scheme files (containing a collection of colors) and template files (instructions concerning how to build the application specific files).
 
 Builders are generally designed for template maintainers' ease of use. Template maintainers SHOULD provide built versions of their template so the end user doesn't need to be aware of the builder.
 
 ## Definitions
 
-A _base16 scheme_ is a YAML file that represents a palette of 16 colors. For
+A _scheme_ is a YAML file which represents a palette of colors. For
 example: the [solarized
 scheme](https://github.com/base16-project/base16-schemes/blob/main/solarized-dark.yaml)
 
-A _base16 template_ is a mustache file that acts as a blueprint; it represents
-how to translate the scheme into an application's desired format. For example: the [vim
+A _template_ is a mustache file which acts as a blueprint; it represents
+how to translate the scheme into an application's desired format. For example: the [base16-vim
 template](https://github.com/base16-project/base16-vim/blob/main/templates/default.mustache)
 is used to convert a _base16 scheme_ into a vim colorscheme.
 
-A _base16 builder_ is an application that implements the full _building_
+A _theme builder_ is an application that implements the full _building_
 feature specification, but MAY include additional functionality. These are
-usually targeted at template maintainers or when building other base16 tooling.
+usually targeted at template maintainers or when building other theme-related tooling.
 
 _Building_ a _template_ is the process of replacing its variables (section 4.1)
 with ones extracted from a _scheme_; usually outputting it to a file, as
-defined by the _template_.
+defined by the _template_ and _template config_.
 
 ## Inputs
 
 ### Schemes Repository
 
-The builder MUST provide a method of loading one or more schemes for use in building templates. The builder MAY provide a method of loading a full directory of schemes at one time. Convenient access to schemes in the [schemes repository](https://github.com/base16-project/base16-schemes) MAY also be provided.
+The builder MUST provide a method of loading one or more schemes for use in building templates. The builder MAY provide a method of loading multiple schemes at one time. Convenient access to schemes in the [schemes repository](https://github.com/base16-project/base16-schemes) MAY also be provided.
 
-This repo contains _scheme files_ for all base16 schemes.
+This repo contains _scheme files_ for all official schemes. We store scheme styles in different folders, but this is not a guarantee.
```

Comment:
"directories" is used commonly in the document so may be better to stick with it.

```suggestion
This repo contains _scheme files_ for all official schemes. We store scheme styles in different directories, but this is not a guarantee.
```